runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -1129,6 +1129,20 @@ else if (event.getMenuOption().equals("Search") && BIRDNEST_IDS.contains(event.g
 			{
 				onInvChange(collectInvItems(LootRecordType.EVENT, BIRDNEST_EVENT));
 			}
+			else if (event.getMenuOption().equals("Clean") && event.getItemId() == ItemID.DIRTY_ARROWTIPS)
+			{
+				// Entire stack gets cleaned at once, want to track how many
+				onInvChange((((invItems, groundItems, removedItems) ->
+				{
+					int itemId = event.getItemId();
```

Comment:
the lifetime of the menu ends after this event is done, and calling getItemId() on it some time later may return the item from a completely different menu